source/extensions/filters/http/rbac/rbac_filter.cc

```diff
@@ -44,7 +44,15 @@ Http::FilterHeadersStatus RoleBasedAccessControlFilter::decodeHeaders(Http::Head
   const Filters::Common::RBAC::RoleBasedAccessControlEngine& engine =
       config_->engine(callbacks_->route());
 
-  if (engine.allowed(*callbacks_->connection(), headers)) {
+  if (engine.allowed(*callbacks_->connection(), headers,
```

Comment:
Probably do this only when there is actually 'darklaunch' policies, otherwise it could be confusing.

Reply:
same thing to 'enforced' policies ? allow by default if no rule in config ?

Reply:
I think probably it's fine to log what user expected to see if they switch the mode ?

Reply:
What mode? This means users will see counter for permissive_allowed all the time, even they do not provide any permissive rules. It seems confusing to me. Anyway, it's just my opinion.